src/figuresClasses.ts

```diff
@@ -1,11 +1,78 @@
-export interface Figure {}
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea: () => number;
+}
+
+function area(areaRound: number): number {
+  return Math.floor(areaRound * 100) / 100;
+}
+
+export class Triangle implements Figure {
+  public shape: Figure['shape'] = 'triangle';
+
+  constructor(
+    public color: Figure['color'],
+    public a: number,
+    public b: number,
+    public c: number,
```

Comment:
Since you're declaring constructor parameters with access modifiers, there's no need to assign them to class properties manually. TypeScript does this implicitly.